sage/vector_store.py

```diff
@@ -222,6 +483,30 @@ def build_vector_store_from_args(args: dict, data_manager: Optional[DataManager]
             alpha=args.retrieval_alpha,
             bm25_cache=bm25_cache,
         )
+    elif args.vector_store_provider == 'chroma':
+        logging.info("Chroma Store is Being Built")
+        print("ChromaDB is instantiated")
```

Comment:
Nit: remove print statement

Reply:
That's outdated. I have removed it actually